src/Controller/EnMarche/UserController.php

```diff
@@ -46,6 +46,7 @@ public function profileDonationAction(DonationRepository $donationRepository): R
         return $this->render('user/my_donation.html.twig', [
             'donations' => $donationRepository->findByEmailAddressOrderedByDonatedAt($userEmail, 'DESC'),
             'subscribed_donations' => $donationRepository->findAllSubscribedDonationByEmail($userEmail),
+            'last_cancel_subscription' => $donationRepository->findLastSubscribedDonationByEmail($userEmail),
```

Comment:
I really don't get the semantic here..

We look for last canceled subscription but repository look for last subscribed donation ?! wtf ?